shared/build.gradle.kts

```
@@ -24,8 +24,13 @@ kmmbridge {
 }
 
 sqldelight {
+    // Read more on gradle options on https://cashapp.github.io/sqldelight/multiplatform_sqlite/gradle/
     database("CioDatabase") {
         packageName = "io.customer.shared.local"
```

```suggestion
        packageName = "io.customer.shared"
```

What is the `.local` for?

---

This is where SQLDelight will auto-generate the classes. I think I missed it and kept what Shahroz added in initial setup. So yes, this needs to be changed, but I think we should keep it in a separate package and not the root one. Since we already have `database` package, how does `io.customer.shared.ciodatabase` sound? Let me know if you have some other name to suggest, I'll then push the changes after moving `sq` file to the relevant package as well so all SQL files are generated under the same package.

---

How about trying `io.customer.shared.database`? I don't see a need to add `cio` in there since it's already `io.customer`.

---

Yes, it certainly is better. But all database related classes we created are already in the same package. We can still use it though, but we might not be able to differentiate between autogenerated and manually created classes through package. If this looks okay, I can update the package to `io.customer.shared.database`

---

I pushed the changes using `io.customer.shared.database`, it looks good to me. Let me know if you have more thoughts on this.